hygieia-jenkins-plugin/src/test/java/hygieia/utils/HygieiaUtilsTest.java

```diff
@@ -1,7 +1,18 @@
 package hygieia.utils;
 
+import com.capitalone.dashboard.misc.HygieiaException;
+import com.capitalone.dashboard.model.BuildStage;
+import com.google.common.io.Resources;
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import org.apache.commons.io.IOUtils;
+import org.assertj.core.api.Assertions;
 import org.junit.Test;
 
+import java.io.IOException;
+import java.util.LinkedList;
+
+import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.Assert.*;
```

Comment:
This is a bad static import which is very expensive

Reply:
@rvema sorry missed that